getting-started/dc2-guide.rst

```diff
@@ -1,165 +1,223 @@
+   butler query-collections /repo/dc2 "u/mrawls/DM-34827*"
 
 These commands should print a list of collections that meet the search criteria.
-The "tree chain" view is a handy way to see the "chained" data products that are included with each collection, such as calibs, skymaps, refcats, etc.
 
 To see some available datasets for processing, try, e.g.,
 
 .. prompt:: bash
 
-   butler query-data-ids /repo/dc2 tract patch visit --collections='2.2i/defaults/test-med-1' --where "skymap='DC2' AND band='g' AND tract=3828 AND patch=47 AND instrument='LSSTCam-imSim'" --datasets "raw"
+   butler query-data-ids /repo/dc2 tract patch visit --collections='u/mrawls/DM-34827/defaults/4patch_4431' --where "skymap='DC2' AND band='g' AND instrument='LSSTCam-imSim'" --datasets "raw"
 
 This command should print a list of data IDs that meet the search criteria, along with their tract, patch, and visit number.
-Certain arguments are required after the ``--where``, including ``skymap`` and ``instrument``, while most others are optional.
+Certain arguments are required after the ``--where``, including ``skymap`` and ``instrument``, while most others are optional, and may include ``band``, ``tract``, ``patch``, etc.
 
-Processing Data
-===============
+Processing Data with the AP Pipelines
+=====================================
 
 Now it's time to process some data.
-In this guide, we will start with raws, run "processCcd" (which includes :py:mod:`lsst.ip.isr.IsrTask`, :py:mod:`lsst.pipe.tasks.characterizeImage.CharacterizeImageTask`, and :py:mod:`lsst.pipe.tasks.calibrate.CalibrateTask`), and make good seeing coadd templates.
+In this guide, we will run a template-building pipeline, ``ApTemplate.yaml``, first.
+This pipeline starts with raw images and runs standard single frame processing (which includes :py:mod:`lsst.ip.isr.isrTask.IsrTask`, :py:mod:`lsst.pipe.tasks.characterizeImage.CharacterizeImageTask`, and :py:mod:`lsst.pipe.tasks.calibrate.CalibrateTask`).
+From here, it is possible to run :py:mod:`lsst.pipe.tasks.postprocess.ConsolidateVisitSummaryTask`, :py:mod:`lsst.pipe.tasks.makeCoaddTempExp.MakeWarpTask`, :py:mod:`lsst.pipe.tasks.selectImages.BestSeeingQuantileSelectVisitsTask`, and :py:mod:`lsst.pipe.tasks.assembleCoadd.CompareWarpAssemblecoaddTask`.
+The final result is good seeing coadd templates.
 
-In a second pipeline, we will run difference imaging using the templates we just built and save the results in an Alert Production Database (APDB).
+In a second pipeline, ``ApPipe.yaml``, we will run difference imaging using the templates we just built.
+This pipeline also starts with single frame processing on raw images, followed by :py:mod:`lsst.ip.diffim.subtractImages.AlardLuptonSubtractTask`, :py:mod:`lsst.ip.diffim.detectAndMeasure.DetectAndMeasureTask`, :py:mod:`lsst.ap.association.transformDiaSourceCatalog.TransformDiaSourceCatalogTask`, and :py:mod:`lsst.ap.association.diaPipe.DiaPipelineTask`.
```

Comment:
It's... complicated. In principle I prefer to have different input science raws, but in practice many folks find it easiest to throw the same pile of raws at both pipelines (that's what we typically do with HSC COSMOS reprocessing, for instance). So I tried to write the guide so that it works either way.